deepspeed/ops/adam/cpu_adam.py

```diff
@@ -96,7 +96,7 @@ def __setstate__(self, state):
             group.setdefault('amsgrad', False)
 
     @torch.no_grad()
-    def step(self, closure=None, fp16_param_groups=None):
+    def step(self, optId=0, closure=None, fp16_param_groups=None):

```

Comment:
this is python land, so camel case is for typically reserved for class names only.

Reply:
so, again, if this case is reduced to only ever having id 0, this is now becoming a bit of a mess. Why do we then create `self.opt_id`, but not use it? i.e. you removed its usage in most places. 

Where does `step()` gets the id now from? It looks like you're relying on the default `0`.

Clearly you guys need to have a bit of a design session around this.

If I may suggestion, let's revert the change you added, so that this PR is just adding a proper cleanup. And then if you decided to kill the counter, this is a different change I'd say.

Reply:
I agree. I revert back my changes